.github/workflows/verify_build_container.yml

```diff
@@ -0,0 +1,20 @@
+name: Build container using the PR
+
+on:
+  pull_request:
+    types: [ opened, synchronize, edited, reopened ]
```

Comment:
Do you know how long the build takes? It might be too much to have this run on every event, perhaps having this triggerable by a comment would be the better option

Reply:
Looking at https://github.com/opendatahub-io/opendatahub-tests/actions/runs/13811294988/job/38633315625?pr=174, it is 1 minute currently. We should validate building the image in pr. It should never break and when it breaks you don't want to find out after merging a PR.

Reply:
please remove `edited`

Reply:
@rnetser I think removing edited defeats the purpose of this, as you'd have a build when the PR is opened but then (potentially) no other builds before it gets merged. With edited at least we'd have a check every time a commit is pushed

Reply:
I agree with @lugi0. Webhook server that we use in many repos, runs this step on every edit. It may seem to be too much, but you don't want to find that build container does not work, after a PR is merged.

Reply:
`edited` also means pr title/description are updated, no? why build on those?

Reply:
btw, the webhook does not build on `edited`, what am i missing?

Reply:
@dbasunag I think @rnetser is right, https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=synchronize#pull_request
```
A pull request's head branch was updated. For example, the head branch was updated from the base branch or new commits were pushed to the head branch.
```
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#pull_request
```
The title or body of a pull request was edited, or the base branch of a pull request was changed.
```

I think we can remove `edited` and keep only `synchronize`

Reply:
webhook, updates anytime the PR is updated (synchronize). I will remove edited.

Reply:
@rnetser and @lugi0 removed `edited`.